pubsublite/spark-connector/spark_streaming_test.py

```diff
@@ -0,0 +1,209 @@
+        }
+    )
+
+    # Create the job config.
+    job = {
+        "placement": {"cluster_name": CLUSTER_ID},
+        "pyspark_job": {
+            "main_python_file_uri": pyfile("spark_streaming_to_pubsublite_example.py"),
+            "jar_file_uris": [
+                "gs://spark-lib/pubsublite/pubsublite-spark-sql-streaming-LATEST-with-dependencies.jar"
+            ],
+            "properties": {"spark.master": "yarn"},
+            "logging_config": {"driver_log_levels": {"root": LoggingConfig.Level.INFO}},
+            "args": [PROJECT_NUMBER, CLOUD_REGION + "-" + ZONE_ID, TOPIC_ID],
+        },
+    }
+
+    operation = job_client.submit_job_as_operation(
+        request={"project_id": PROJECT_ID, "region": CLOUD_REGION, "job": job}
+    )
+    response = operation.result()
+
+    # Dataproc job output gets saved to the Google Cloud Storage bucket
+    # allocated to the job. Use a regex to obtain the bucket and blob info.
+    matches = re.match("gs://(.*?)/(.*)", response.driver_output_resource_uri)
+
+    output = (
+        storage.Client()
+        .get_bucket(matches.group(1))
+        .blob(f"{matches.group(2)}.000000000")
+        .download_as_text()
+    )
+
+    assert (
+        "INFO com.google.cloud.pubsublite.spark.PslStreamWriter: Committed 1 messages for epochId"
+        in output
+    )
+
+
+def test_spark_streaming_from_pubsublite(
+    topic: Topic, subscription: Subscription
+) -> None:
+    from google.cloud.dataproc_v1.types import LoggingConfig
+
+    # Create a Dataproc job client.
+    job_client = dataproc_v1.JobControllerClient(
+        client_options={
+            "api_endpoint": "{}-dataproc.googleapis.com:443".format(CLOUD_REGION)
+        }
+    )
+
+    # Create the job config.
+    job = {
+        "placement": {"cluster_name": CLUSTER_ID},
+        "pyspark_job": {
+            "main_python_file_uri": pyfile(
+                "spark_streaming_from_pubsublite_example.py"
+            ),
+            "jar_file_uris": [
+                "gs://spark-lib/pubsublite/pubsublite-spark-sql-streaming-LATEST-with-dependencies.jar"
+            ],
+            "properties": {"spark.master": "yarn"},
+            "logging_config": {"driver_log_levels": {"root": LoggingConfig.Level.INFO}},
+            "args": [PROJECT_NUMBER, CLOUD_REGION + "-" + ZONE_ID, SUBSCRIPTION_ID],
+        },
+    }
+
+    operation = job_client.submit_job_as_operation(
+        request={"project_id": PROJECT_ID, "region": CLOUD_REGION, "job": job}
+    )
+    response = operation.result()
+
+    # Dataproc job output gets saved to the Google Cloud Storage bucket
+    # allocated to the job. Use a regex to obtain the bucket and blob info.
+    matches = re.match("gs://(.*?)/(.*)", response.driver_output_resource_uri)
+
+    output = (
+        storage.Client()
+        .get_bucket(matches.group(1))
+        .blob(f"{matches.group(2)}.000000000")
+        .download_as_text()
+    )
+
+    assert "Batch: 0\n" in output
```

Comment:
I would add more here for better coverage, like https://github.com/googleapis/java-pubsublite-spark/blob/master/samples/snippets/src/test/java/pubsublite/spark/SamplesIntegrationTest.java#L254-L308, since it's fixed, we should be able to validate fixed output.

Reply:
Adjusted the test to look for fixed output.